R/qra_fit_convenience.R

```diff
@@ -365,6 +200,8 @@ load_covid_forecasts_relative_horizon <- function(
 #' @param forecast_week_end_date date relative to week-ahead or day-ahead
 #' targets are defined. For week ahead targets, a Saturday; for day ahead
 #' targets, a Monday.
+#' @param max_horizon maximum horizon relative to forecast_week_end_date and 
+#' analogous training reference dates for both training and prediction
```

Comment:
Main point of this suggestion is to avoid the `forecast_week_end_date` terminology, but also I think the description of intent can be simplified for the purpose of this function
```suggestion
#' @param max_horizon maximum horizon for ensemble forecasts
```